src/avatar.cpp

```diff
@@ -750,6 +774,18 @@ bool avatar::read( int inventory_position, const bool continuous )
                  complex_player->disp_name() );
     }
 
+
+    // push an indentifier of martial art book to the action handling
+    if( it.type->use_methods.find( "MA_MANUAL" ) != it.type->use_methods.end() ) {
+
+        if( g->u.stamina < g->u.get_stamina_max() * 0.1 ) {
```

Comment:
```suggestion
        if( g->u.stamina < g->u.get_stamina_max() / 10 ) {
```
This magic value of "10% of stamina" has appeared in two places now, so it should probably be pulled out somewhere and given a name.